### DISCUSSION TITLE
Which kind of stream is the video created by motion?

### DISCUSSION THREAD

**Discussion by hwypengsir** [Motion-Project/motion#1705] [Q&A]
We can watch the video in browser `127.0.0.1:8081` created by motion,which kind of stream is the video created by motion?It is a rtsp stream?

**Comment by Mr-Dave** [comment]
Motion provides a MJPEG stream not RTSP.  Motionplus (a different application under the Motion-Project) just recently added the option for a MPEGTS stream.
